curl_cffi/requests/session.py

```diff
@@ -85,14 +81,8 @@ def has(cls, item):
 
     @classmethod
     def normalize(cls, item):
-        if item == "chrome":
-            return cls.chrome
-        elif item == "safari":
-            return cls.safari
-        elif item == "safari_ios":
-            return cls.safari_ios
-        else:
-            return item
+        item_map = {"chrome": cls.chrome, "safari": cls.safari, "safari_ios": cls.safari_ios}
+        return item_map.get(item, item)
```

Comment:
This function is also called quite frequently, I prefer to keep the if-else style instead of table-driven style. Perhaps, the `match` expression is equivlent to if-else, but that available in later versions.